app/assets/javascripts/discourse/tests/helpers/qunit-helpers.js

```diff
@@ -71,7 +71,12 @@ import {
 } from "discourse/lib/to-markdown";
 
 export function currentUser() {
-  return User.create(sessionFixtures["/session/current.json"].current_user);
+  const user = User.create(
+    sessionFixtures["/session/current.json"].current_user
+  );
+  user.appEvents = getOwner(this).lookup("service:appEvents");
+  user.trackStatus();
```

Comment:
Are we stopping the tracking somewhere in tests?

Reply:
I've moved status tracking to the `acceptance()` test helper and added stopping in `afterEach(), thank you.

Reply:
No problem :+1: